Services/CO.CDP.DataSharing.WebApi.Tests/UseCase/GetSharedDataUseCaseTest.cs

```diff
@@ -115,10 +117,11 @@ private void AssertAdditionalIdentifiers(IEnumerable<Identifier>? additionalIden
         additionalIdentifiers.Should().NotBeNull();
         additionalIdentifiers.Should().HaveCount(1);
 
+        var scheme = "GB-COH";
         var identifier = additionalIdentifiers?.First();
-        identifier?.Scheme.Should().Be("GB-COH");
+        identifier?.Scheme.Should().Be(scheme);
         identifier?.LegalName.Should().Be("AnotherLegalName");
-        identifier?.Uri.Should().Be(new Uri("http://example.com"));
+        identifier?.Uri.Should().Be(IdentifierSchemes.GetRegistryUri(_configurationService.Object.GetOrganisationApiHostUrl(), scheme, identifier.Id));
```

Comment:
`configurationService.Object.GetOrganisationApiHostUrl()` is never set up, so it always returns an empty string? It's also not passed anywhere, so isn't actually used by code under test, or am i missing something?